### PR TITLE
Judicial-system/Added gender and defender info to overview screen for prosecutors and judges

### PR DESCRIPTION
**Pull request by oddsson**
# Added gender and defender info to overview screen for prosecutors and judges

https://app.asana.com/0/322488613152836/1199172915726483
https://app.asana.com/0/322488613152836/1199133898226070

## What

See ☝️

## Why

For clarity

### Prosecutor

screencapture-localhost-4200-stofna-krofu-yfirlit-030000e8-6254-4e50-b87b-cf153abaf99b-2020-11-16-09_07_55

### Judge

screencapture-localhost-4200-domari-krafa-030000e8-6254-4e50-b87b-cf153abaf99b-2020-11-16-09_10_11

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by oddsson** [comment]
Waiting for #1902 to be able to reuse a function created there

**Review by addi — DISMISSED** [review]

File: apps/judicial-system/web/src/utils/stepHelper.tsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import {
   Case,
   CaseAppealDecision,
   CaseCustodyRestrictions,
+  CaseGender,
+  Gender,
```
Comment (gudjong) [inline apps/judicial-system/web/src/utils/stepHelper.tsx:10]: CaseGender and Gender are not used in this file. Please remove.

File: apps/judicial-system/web/src/utils/utils.spec.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {
   CaseTransition,
   CaseCustodyRestrictions,
   Case,
+  CaseGender,
```
Comment (gudjong) [inline apps/judicial-system/web/src/utils/utils.spec.ts:14]: CaseGender is not used in this file. Please remove.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/utils/stepHelper.tsx (hunk 1 above)
Comment (oddsson) [reply apps/judicial-system/web/src/utils/stepHelper.tsx]: Good eye, thanks 🙏

**Review by gudjong — APPROVED** [review]

**Pull request merged**